browser/model/jbds-autoinstall.js

```diff
@@ -12,7 +12,7 @@ class JbdsAutoInstallGenerator {
   }
 
   generate(jbdsInstallDir, jdkInstallDir) {
-    let jbdsVersion = "9.1";
+    let pjson = require('../../package.json');

```

Comment:
You could also scrape the HTML page and get this information. 

https://devstudio.jboss.com/10.0/snapshots/builds/devstudio.product_master/latest/all/ -> jboss-devstudio-10.0.0.Alpha1-v20160506-2012-B5309-installer-standalone.jar